core/storage/src/storages/mod.rs

```diff
@@ -195,6 +207,19 @@ where
         )
     }
 
+    #[inline]
+    pub fn set_best_block_index(&mut self, value: u64) -> io::Result<()> {
+        let mut key = self.enc_slice(self.genesis_hash.expect(MISSING_GENESIS_ERR).as_ref());
+        key.extend_from_slice(&[0]);
```

Comment:
Why this is line necessary?

Reply:
I'm using the same column for best hash and best index, so storing index at `genesis_hash` + [0]

Reply:
I see. Why not making a separate column?

Reply:
Just feels weird having columns with 1 value in them. Generally the design of how we storing this values seems imperfect - we have a hashmap of genesis hash to best_hash <-- e.g. there always going to be one value there.

Reply:
That's just a limitation of a columnio storage. If we want to store some miscellaneous one-off data like genesis and best index we can keep them in a single meta column.